dockers/docker-dhcp-relay/port-name-alias-map.txt.j2

```diff
@@ -0,0 +1,5 @@
+{# Generate port name-alias map for isc-dhcp-relay to parse. Each line contains one #}
+{#  name-alias pair of the form "<name> <alias>" #}
+{% for port, config in PORT.items() %}
+    {{- port }} {% if "alias" in config %}{{ config["alias"] }}{% else %}{{ port }}{% endif %} {{ "\n" -}}
```

Comment:
It looks like an extra space will be generated at at the end of each line due to the space before `{{ "\n" -}}`. If so, please remove.

Reply:
done